core/services/functions/external_adapter_client.go

```diff
@@ -122,7 +139,7 @@ func (ea *externalAdapterClient) RunComputation(
 		Data:                &data,
 	}
 
-	userResult, userError, domains, err = ea.request(ctx, payload, requestId, jobName)
+	userResult, userError, domains, err = ea.request(ctx, payload, requestId, jobName, "request")
```

Comment:
Could we change the label "request" to something more descriptive (and easily searchable) for the logs? Maybe "computation_request"?